### PR TITLE
Fix/event cards

### PR DESCRIPTION
**Pull request by DiD3n**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "@typescript-eslint/parser": "^4.31.2",
     "axios": "^0.23.0",
     "classnames": "^2.3.1",
+    "dayjs": "^1.11.3",
```
Comment (JUtupe) [inline package.json:92]: *sad momentjs noises in distance*

File: src/api/graphql/events/EventDataQuery.ts [hunk 2]
```diff
@@ -29,6 +33,11 @@ export interface EventQueryData {
     event: EventData
 }
 
+export interface EventLocationData {
```
Comment (JUtupe) [inline src/api/graphql/events/EventDataQuery.ts]: I would suggest using name `EventAddressData` here

File: src/common/models/EventLocation.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        public city: string,
+        public latitude?: number,
+        public longitude?: number
+    ) {
+    }
+
+    static fromData(data: EventLocationData): EventLocation {
+        return new EventLocation(
+            data.place,
+            data.city
```
Comment (JUtupe) [inline src/common/models/EventLocation.ts]: coordinates?

pulp-fiction-john-travolta

File: src/components/containers/EventCard/EventCard.module.scss [hunk 4]
```diff
@@ -5,15 +5,44 @@
...
+    background-size: 51%, 50%;
+    background-repeat: no-repeat;
+
     box-sizing: border-box;
-    position: relative;
 
     margin: 8px 0;
     min-height: 113px;
-    padding: var(--padding) var(--padding) 48px;
+    padding: 14px;
```
Comment (JUtupe) [inline src/components/containers/EventCard/EventCard.module.scss:15]: use variable here

File: src/components/containers/EventCard/EventCard.module.scss [hunk 5]
```diff
@@ -5,15 +5,44 @@
...
 
     border-radius: 16px;
     box-shadow: 0 2px 24px rgba(7, 16, 24, 0.3);
+
+    .title {
+        margin: 0;
+        font-size: 24px;
```
Comment (JUtupe) [inline src/components/containers/EventCard/EventCard.module.scss]: font-size should be grouped with font related attributes that are lower

File: src/components/containers/EventCard/chips/StartDateChip.tsx [hunk 6]
```diff
@@ -1,16 +1,22 @@
...
\ No newline at end of file
+export const StartDateChip: React.FC<StartDateChipProps> = ({date}) => {
+    const startDate = dayjs(date)
+
+    return (
+        <div className={styles.chip}>
+            <ClockIcon/>
+            <span className={styles.text}>
+                {/* display date with year only if year is different for current*/}
+                {startDate.format(`D MMMM ${startDate.year() !== dayjs().year() ? 'YYYY' : ''}, HH:mm`)}
```
Comment (JUtupe) [inline src/components/containers/EventCard/chips/StartDateChip.tsx]: should be extracted to some helper function

File: src/components/containers/EventCard/EventCard.tsx [hunk 7]
```diff
@@ -22,25 +24,22 @@ export const EventCard: React.FC<EventCardProps> = ({
     startDate,
     locationName,
 }) => {
+    // color fallback
+    if (!color) color = "#4340BE"
```
Comment (JUtupe) [inline src/components/containers/EventCard/EventCard.tsx:27]: Can you extract this color to consts or somewhere else?

**Review by JUtupe — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (DiD3n) [reply package.json:92]: yeah... sure 🤡
image

File: src/api/graphql/events/EventDataQuery.ts (hunk 2 above)
Comment (DiD3n) [reply src/api/graphql/events/EventDataQuery.ts]: seems reasonable

File: src/components/containers/EventCard/EventCard.module.scss (hunk 4 above)
Comment (DiD3n) [reply src/components/containers/EventCard/EventCard.module.scss:15]: i removed this variable, 
after rework it was used in only one place

File: src/components/containers/EventCard/EventCard.tsx (hunk 7 above)
Comment (DiD3n) [reply src/components/containers/EventCard/EventCard.tsx:27]: i dunno, we don't have any kind of project pallet yet
so making one, when some thing are going to change seems not worth effort (in this case)

File: src/components/containers/EventCard/chips/StartDateChip.tsx (hunk 6 above)
Comment (DiD3n) [reply src/components/containers/EventCard/chips/StartDateChip.tsx]: i moved it above the jsx return i don't think it's necessary to separate this from component
it won't be used anywhere else anyways

File: src/components/containers/EventCard/EventCard.tsx (hunk 7 above)
Comment (JUtupe) [reply src/components/containers/EventCard/EventCard.tsx:27]: ok then

**Review by JUtupe — APPROVED** [review]

**Pull request merged**